### ISSUE TITLE
Matrix distorted while printing

### ISSUE THREAD

**Issue by JoopStringer** [surveyjs/survey-analytics#192]
Hi all,

I have a matrix of questions and answers and they get distorted while printing.
Any idea, suggestion, solution ?
You can see in this picture on the bottom the matrix and on top the print ...

<img width="970" alt="Schermafbeelding 2022-01-18 om 20 12 45" src="https://user-images.githubusercontent.com/688083/150003441-7224aab2-6f59-45ab-bf25-a9fd13b257ed.png">

**Comment by tsv2013** [comment]
Are you asking about export to PDF functionality?

**Comment by JoopStringer** [comment]
No, this happens when I try to print a page in ViewOnly mode (either on paper or save as PDF)
So question is what happens to the matrix in print, is there a special CSS which gets loaded ??

**Comment by tsv2013** [comment]
Are you trying to print survey via browser print dialog?

**Comment by JoopStringer** [comment]
@tsv2013 to answer your question: in the screenprint YES, but only to show the difference

In my system I use a PDF generator to save the outcome ... I noticed this discrepancy between viewing it on screen and "printing". So question is ... what happens to the matrix if it is printed/saved to PDF.
It was working in version 1.7.28 (i believe)

Now I use the latest version:
<link href="https://unpkg.com/survey-jquery/modern.min.css" type="text/css" rel="stylesheet">
<script type="text/javascript" src="https://unpkg.com/survey-jquery"></script>
